src/libs/elektra/kdb.c

```diff
@@ -831,12 +851,28 @@ int kdbGet (KDB * handle, KeySet * ks, Key * parentKey)
 static int elektraSetPrepare (Split * split, Key * parentKey, Key ** errorKey, Plugin ** hooks)
 {
 	int any_error = 0;
+
+	// GLOBAL: setresolver [init]
+	// GLOBAL: setresolver [max once]
+
+	// GLOBAL: presetstorage [init]
+	// GLOBAL: presetstorage [max once]
```

Comment:
Seems like we need to split the loop to have presetstorage and setstorage separated.

setresolver would be the COMMIT_PLUGIN, its not within this loop.